app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,67 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  contact: null,
+  tags: [],
 };
 
+export const ASSIGNED_CONTACT = [
+  {
+    label: "Admin",
+    value: "Admin",
+  },
+  {
+    label: "Non Admin",
+    value: "Non_Admin",
+  },
+];
+
+export const TAGS = [
+  {
+    label: "Geeting Started",
+    value: "Getting_Started",
+  },
+  {
+    label: "Meeting",
+    value: "Meeting",
+  },
+  {
+    label: "Learning",
+    value: "Learning",
+  },
+  {
+    label: "Onboarding",
+    value: "Onboarding",
+  },
+  {
+    label: "Leaves and Holidays",
+    value: "Leaves_and_Holidays",
+  },
+];
```

Comment:
Go through the Slack thread. Let's build these values using a `buildSelectOptions` util.